tests/components/ActionButton/ToggleActionButton.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { SinonSandbox, createSandbox, spy, SinonSpy } from 'sinon';
+import { Mock } from 'coveo-search-ui-tests';
+import { IToggleActionButtonOptions, ToggleActionButton } from '../../../src/components/ActionButton/ToggleActionButton';
+import * as icons from '../../../src/utils/icons';
+import { ActionButton } from '../../../src/components/ActionButton/ActionButton';
+
+describe('ToggleActionButton', () => {
+    let sandbox: SinonSandbox;
+    let options: IToggleActionButtonOptions;
+    let testSubject: ToggleActionButton;
+
+    let clickSpy: SinonSpy;
+    let activateSpy: SinonSpy;
+    let deactivateSpy: SinonSpy;
+    let updateIconSpy: SinonSpy;
+    let updateTooltipSpy: SinonSpy;
+
+    beforeAll(() => {
+        sandbox = createSandbox();
+
+        clickSpy = spy();
+        activateSpy = spy();
+        deactivateSpy = spy();
+        updateIconSpy = spy(<any>ActionButton.prototype, 'updateIcon');
+        updateTooltipSpy = spy(<any>ActionButton.prototype, 'updateTooltip');
```

Comment:
Why are you not using the sandbox?

Reply:
You're totally right. It should use `sandbox` there.